docker/docker-compose.dev.yml

```diff
@@ -0,0 +1,15 @@
+version: '3'
+
+services:
+  web:
+    build:
+      context: ..
+      dockerfile: docker/Dockerfile.dev
```

Comment:
Can we match https://github.com/libero/browser/blob/master/Dockerfile? So `prod` -> `test` -> `dev`.

Reply:
If I'm reading this correctly, all of these images will be created when referencing this file. Is that correct? I don't see the advantage of having everything in one file. Could you explain please?

Reply:
Could leave the `test` and `prod` image for later when we [setup a pipeline](https://github.com/libero/libero/issues/96)

Reply:
Ok. Was just bit concerned that this looked like it would be a `dev`-only container.

Reply:
It is at the moment (doesn't have the code), but it should be transformed into a `prod` container derivative as soon as the `prod` container image is there